src/app/legacy/psammead/gel-foundations/src/prop-types/index.js

```diff
@@ -1,3 +1,3 @@
 /* eslint-disable import/prefer-default-export */
 
-export scriptPropType from './scripts';
+export { default as scriptPropType } from './scripts';
```

Comment:
This was angry... In order to properly export and export, they needed to be wrapped in `defaults`'s.